### PR TITLE
Setup trendy highlights

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
Hi,

- When the page loads, the webapp shows the exchange rate of the list of countries against a base currency.
- Along with the currency name the exchange rate of that currency against base currency has been displayed.
- When the user clicks (or taps) on a item, the application navigates to the details page.
- In the details page, the webapp retrieves data from the API to show detailed data .
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- created test using react testing library.

Loom video : 
https://www.loom.com/share/806e5119b8a84043a25922e72cb27614

Live Demo: https://trendy-highlights.onrender.com/

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,197 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my morning session teammates who gave me suggestions regarding this project. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```
Comment (iambenkis) [inline README.md]: - [x] Good job adding the README that contains an "Acknowledgements" section 👏 , however you should gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly  add in your  "Acknowledgements" section  the  appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👨‍🍳

File: src/tests/BaseCard.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  );
+  it('Should render the content correctly', () => {
+    render(wrap(base));
+    expect(screen.getByText('USD')).not.toBeNull();
+  });
+
+  it('Should match the snapshot', () => {
+    expect(renderer.create(wrap(base)).toJSON()).toMatchSnapshot();
+  });
+});
```
Comment (iambenkis) [inline src/tests/BaseCard.test.js:34]: - [x] Good job setting up the testes 👍 , however all the testes you set up are not passing. Kindly ensure that all your testes are passing as you know testes are very important when you are developing a product, they will help you to debug in the future 😃 

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sajibarafatsiddiqui  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Good job presenting your project in the above video ,however your video presenation should contain :
1) The features of the project.
2) A highlight about a piece of the code the student finds interesting or is proud of making.
These two things miss in your video kindly re-record the video with a complete requirement 😄

 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply README.md]: Added accomplishment. Thanks.

File: src/tests/BaseCard.test.js (hunk 2 above)
Comment (sajibarafatsiddiqui) [reply src/tests/BaseCard.test.js:34]: I checked all the tests. For a silly mistake, one failed. I fixed it. Thanks.

**Comment by sajibarafatsiddiqui** [comment]
Hi I created new video discussing about my coding in this project and the features.

Here is the link:
https://www.loom.com/share/66ea145f75184c4b820b0815f2852ff8
Thanks

File: src/tests/BaseCard.test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    />
+  );
+  const baseCurrency = 'USD';
+
+  const wrap = (component) => (
+    <Provider store={store}>
+      <BaseCurrencyContext.Provider value={{ baseCurrency }}>
+        <Router>
+          {component}
+        </Router>
```
Comment (iambenkis) [inline src/tests/BaseCard.test.js:22]: - [x] Keep doing the great work 👏

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hello @sajibarafatsiddiqui , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️

- [x] Good job presenting your project in the above video and all the request changes given previously, but your video should have no more than five minutes , kindly fix then, and you'll be ready to go to the next step 👍 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @iambenkis  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/BaseCard.test.js (hunk 3 above)
Comment (sajibarafatsiddiqui) [reply src/tests/BaseCard.test.js:22]: Hi @iambenkis I have created another video of less than 5 minutes.
loom video:https://www.loom.com/share/8d6957a97e28410d8eb40092b404df68

Thanks

**Review by AnselemOdims — APPROVED** [review]
### Hi  @sajibarafatsiddiqui 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**